lizmap/modules/lizmap/lib/Request/Proxy.php

```diff
@@ -385,10 +388,27 @@ protected static function sendRequest($url, $options)
             $options['headers']['Referer'] = $options['referer'];
         }
 
+        // Create request cache strategy
+        $strategy = new Strategy\Delegate\DelegatingCacheStrategy($defaultStrategy = new Strategy\NullCacheStrategy());
+        $lizmapServices = self::getServices();
+        $qgisServerUrl = rtrim($lizmapServices->wmsServerURL, '/');
+        if (!empty($lizmapServices->lizmapPluginAPIURL)) {
+            $qgisServerUrl = rtrim($lizmapServices->lizmapPluginAPIURL, '/');
+        }
```

Comment:
See PR #5266 to avoid code duplication
I will update it, I just realized the mistake.

Reply:
yes it is still a draft, I have planned to do #5266